docker_files_test.go

```diff
@@ -19,14 +19,10 @@ func TestCopyFileToContainer(t *testing.T) {
 
 	// copyFileOnCreate {
 	absPath, err := filepath.Abs(filepath.Join(".", "testdata", "hello.sh"))
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
```

Comment:
suggestion: sorry the intent was to only use require for the new test, all these changes will cause a bunch of conflicts with other PR's which are inflight, so would we revert all of these and just use it for your new test.

Sorry to be be a pain, but this would cause quit a lot of conflicts.

Reply:
Cool. Made the changes.